include/znc/Utils.h

```diff
@@ -185,9 +185,10 @@ class CTable : protected std::vector<std::vector<CString>> {
     /** Get a line of the table's output
      *  @param uIdx The index of the line you want.
      *  @param sLine This string will receive the output.
+     *  @param as_deflist (optional) Opt into ChanServ style output (two-columns only)
      *  @return True unless uIdx is past the end of the table.
      */
-    bool GetLine(unsigned int uIdx, CString& sLine) const;
+    bool GetLine(unsigned int uIdx, CString& sLine, bool as_deflist = false) const;
```

Comment:
GetLine is always called though a wrapper that prints it out in one go, and since the change is required in there, that's where I put it. 

I think you might like my [second solution for this](/girst/znc/tree/set-table-style) better, then?